eva/executor/create_mat_view_executor.py

```diff
@@ -0,0 +1,87 @@
+# coding=utf-8
+# Copyright 2018-2020 EVA
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from eva.catalog.catalog_manager import CatalogManager
+from eva.planner.create_mat_view_plan import CreateMaterializedViewPlan
+from eva.planner.types import PlanOprType
+from eva.executor.abstract_executor import AbstractExecutor
+from eva.utils.generic_utils import generate_file_path
+from eva.storage.storage_engine import StorageEngine
+from eva.expression.abstract_expression import ExpressionType
+
+from eva.utils.logging_manager import LoggingManager, LoggingLevel
+
+
+class CreateMaterializedViewExecutor(AbstractExecutor):
+
+    def __init__(self, node: CreateMaterializedViewPlan):
+        super().__init__(node)
+
+    def validate(self):
+        pass
+
+    def exec(self):
+        """Create materialized view executor
```

Comment:
Can we move the create table part to create_executor.py?

Reply:
Done